kafka/proxy_test.go

```diff
@@ -53,12 +63,12 @@ func TestRequestKeyHandler_Handle(t *testing.T) {
 	}{
 		{
 			name:        "Valid message",
-			request:     []byte{0, 0, 0, 7, 0, 16, 99, 111, 110, 115, 111, 108, 101, 45, 112, 114, 111, 100, 117, 99, 101, 114, 255, 255, 0, 1}, // payload: 'valid message'
+			request:     []byte{0, 0, 5, 220, 0, 0, 0, 1, 0, 4, 100, 101, 109, 111, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 81, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 69, 255, 255, 255, 255, 2, 42, 190, 231, 201, 0, 0, 0, 0, 0, 0, 0, 0, 1, 122, 129, 58, 51, 194, 0, 0, 1, 122, 129, 58, 51, 194, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 0, 0, 0, 1, 38, 0, 0, 0, 1, 26, 118, 97, 108, 105, 100, 32, 109, 101, 115, 115, 97, 103, 101, 0}, // payload: 'valid message'
```

Comment:
Could we provide the exact request body in tests and then convert it to bytes?  At the moment I don't see which message is valid/invalid.

Reply:
We could, but at this moment we do not have a proper encoder. 
Encoder/Decoder code comes from Sarama library and at this stage I only copied (yes, I had to copy because their interfaces are not public) the decoder logic, which is the only thing we need in our app by now.

I would postpone the encoding logic for later once the project turns to a more mature stage. There will be several request headers and message versions to support.

Reply:
Ok, I understand, but you can extend the request body as comment with a "shape" of valid/invalid message? Then it will be easy to check if tests passed or not by changes or by bug in changes 😅

Reply:
I dont get what you mean.

Reply:
@magicmatatjahu Look at my last commit. Not sure if this is what you meant but I hope it looks more clear now.
Ideally, we would be doing this through the ProduceRequest struct, but the cons are all of the ones I mentioned in my previous message.

Reply:
Sorry, I have this situation 😅 

![image](https://user-images.githubusercontent.com/20404945/124905984-e4db9500-dfe6-11eb-9a76-ad6b6ad405ac.png)

So yeah, the comment was for describing the shape of bytes. Ok, we can stay with the last commit :)

Reply:
So please fix the lint error and I will accept.

Reply:
Nah, I ended up creating a very handmade encoder that only works for tests. Enjoy checking the bytes back and forth dance 😆 . Btw, I ended up using some "magic" bytes so I simplify the work, otherwise, I will be spending a lot of time.

Reply:
Now I feel responsible for your efforts. I only had problem with these bytes and I don't like stop you (because it's only a test), so I accept changes 🚀  Good job!